tests/conftest.py

```diff
@@ -9,22 +9,23 @@
 import requests_mock as _requests_mock
 
 from homeassistant import util
-from homeassistant.util import location
 from homeassistant.auth.const import GROUP_ID_ADMIN, GROUP_ID_READ_ONLY
-from homeassistant.auth.providers import legacy_api_password, homeassistant
+from homeassistant.auth.providers import homeassistant, legacy_api_password
+from homeassistant.util import location
 
-pytest.register_assert_rewrite("tests.common")
```

Comment:
This was above `tests.common` on purpose. Although I don't know why. 

https://docs.pytest.org/en/latest/writing_plugins.html#assertion-rewriting

Reply:
```
tests/conftest.py:28
  /Users/paulus/dev/hass/home-assistant/tests/conftest.py:28: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: tests.common
    pytest.register_assert_rewrite("tests.common")
```

Reply:
What should we do now?

Reply:
Well, we going to restore that piece with an isort ignore added to it 👍

Reply:
I have opened a PR that returns this to its old state, see https://github.com/home-assistant/home-assistant/pull/29797.

Reply:
Darn you are quick, was just finishing up a PR 😆 
Nevertheless, thanks @basnijholt! 👍